docs/user-guide/private-repositories.md

```diff
@@ -29,7 +45,10 @@ Instead of using username and password you might use access token. Following ins
 * [Gitlab](https://docs.gitlab.com/ee/user/project/deploy_tokens/)
 * [Bitbucket](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
 
-Then, connect the repository using an empty string as a username and access token value as a password.
+Then, connect the repository using any non-empty string as username and the access token value as a password. 
+
+!!!note
```

Comment:
check syntax against https://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
It reads:

> Admonition blocks follow a simple syntax: every block is started with !!!, followed by a single keyword which is used as the type qualifier of the block. The content of the block then follows on the next line, indented by four spaces.

I guess the whitespace between `!!!` and the keyword is optional, because the output of `mkdocs` shows correcly:

![image](https://user-images.githubusercontent.com/3942683/62495170-6dd36b80-b7d5-11e9-9232-d416df2ca8c4.png)